docs/reference/release-notes/7.16.asciidoc

```diff
@@ -0,0 +1,434 @@
+* Add ability to augment classes with fields from other classes in Painless {es-pull}76628[#76628]
+* Add dynamic (duck) type resolution to Painless static types {es-pull}78575[#78575]
+* Adds a lookup method to Painless for finding methods of all sub classes {es-pull}77044[#77044]
+* Script: `UnsignedLong` field type converter {es-pull}77271[#77271]
+* Script: compile/cache eviction history metric placeholders {es-pull}78257[#78257] (issue: {es-issue}62899[#62899])
+
+Infra/Settings::
+* Add 'show' command to the keystore CLI {es-pull}76693[#76693] (issue: {es-issue}57261[#57261])
+
+Ingest::
+* Add enrich node cache {es-pull}76800[#76800] (issue: {es-issue}48988[#48988])
+* Add indices pipeline settings check when deleting a pipeline {es-pull}77013[#77013]
+* Allow range types to be used for enrich matching {es-pull}76110[#76110]
+* ECS support for Grok processor {es-pull}76885[#76885] (issue: {es-issue}66528[#66528])
+* Improving cache lookup to reduce recomputing / searches {es-pull}77259[#77259]
+* Make enrich policy execution cancelable {es-pull}77188[#77188] (issue: {es-issue}48988[#48988])
+* Optimistic concurrency control for updating ingest pipelines {es-pull}78551[#78551] (issue: {es-issue}77031[#77031])
+* Sync grok processor patterns with Logstash {es-pull}76752[#76752]
+* Updating ingest pipeline without changes is no-op {es-pull}78196[#78196] (issue: {es-issue}77382[#77382])
+
+Java High Level REST Client::
+* Add support for rest compatibility headers to the HLRC {es-pull}78490[#78490] (issue: {es-issue}77859[#77859])
+
+License::
+* Add license family attribute to feature usage tracking {es-pull}76622[#76622]
+* Add utility for tracking licensed persistent tasks {es-pull}76672[#76672]
+* Separate feature usage tracking for FLS and DLS {es-pull}79152[#79152]
+* Use a licensed feature per realm-type (+custom) {es-pull}78810[#78810]
+
+Machine Learning::
+* Add new `normalize_above` parameter to `p_value` significant terms heuristic {es-pull}78833[#78833]
+* Add new default char filter `first_line_with_letters` for machine learning categorization {es-pull}77457[#77457]
+* Adding new `defer_definition_decompression` parameter to put trained model API {es-pull}77189[#77189] (issue: {es-issue}77132[#77132])
+* Enable ML on macOS on ARM {es-pull}78203[#78203]
+* Track feature usage for data frame analytics, inference, and anomaly jobs {es-pull}76789[#76789]
+
+Mapping::
+* Better error message for long keys in flattened fields {es-pull}80433[#80433] (issue: {es-issue}78248[#78248])
+* Add `time_series_metric` parameter {es-pull}76766[#76766] (issue: {es-issue}74014[#74014])
+* Add dimension mapping parameter {es-pull}74450[#74450] (issue: {es-issue}74014[#74014])
+
+Monitoring::
+* Remove license check for monitoring data retention {es-pull}79010[#79010]
+
+Other::
+* Fleet: Add `action_response` into .fleet-actions-results mapping {es-pull}79584[#79584]
```

Comment:
This looks to update an index template for a Fleet system data stream. I think it's a better fit under `Indices APIs`.
```suggestion
* Fleet: Add `action_response` into `.fleet-actions-results` mapping {es-pull}79584[#79584]
```